pkg/accounts/adaptor/controller/account.ts

```diff
@@ -438,4 +475,100 @@ export class AccountController {
       }),
     );
   }
+
+  async setAvatar(
+    targetAccountName: string,
+    actorID: string,
+    medium: string,
+  ): Promise<Result.Result<Error, void>> {
+    const accountRes = await this.fetchService.fetchAccount(
+      targetAccountName as AccountName,
+    );
+    if (Result.isErr(accountRes)) {
+      return accountRes;
+    }
+    const account = Result.unwrap(accountRes);
+
+    const res = await this.avatarService.create(
+      account.getID(),
+      medium as MediumID,
+      actorID as AccountID,
+    );
+    if (Result.isErr(res)) {
+      return res;
+    }
+
+    return Result.ok(undefined);
```

Comment:
```suggestion
    return res;
```
`res` は `Result<Error, void>` 型なので, そのまま返してよいと思います.
ヘッダーの方と `unset` する方の処理も同様です.